Description: Change the manual page to show the Debian-specific installation
 paths of supplementary files. Upstream strived to justify his refusal to
 convert the package to autoconf or to employ some other installation-time
 substitution.
Author: Laszlo Ersek <lacos@caesar.elte.hu>
Forwarded: http://lists.debian.org/debian-mentors/2009/02/msg00141.html
Reviewed-by: Paul Wise <pabs@debian.org>
Last-Update: 2009-10-28
Index: lbzip2/lbzip2.1
===================================================================
--- lbzip2.orig/lbzip2.1	2009-10-20 23:06:16.000000000 +0200
+++ lbzip2/lbzip2.1	2009-10-20 23:15:38.000000000 +0200
@@ -146,8 +146,8 @@
 .B lbzip2
 prints a memory allocation trace to standard error. The trace can be checked
 with the
-.B malloc_trace.pl
-Perl script (part of the source distribution).
+.B /usr/share/lbzip2/malloc_trace.pl
+Perl script.
 
 .SH OPERANDS
 .TP
@@ -241,9 +241,10 @@
 either by default or by manual selection) doesn't check stream-level (combined)
 CRCs, only block-level CRCs. It can also fail to decompress some rare valid
 .B .bz2
-files. (See the Bugs section of the README in the source distribution for more
-on this.) Since the author cannot rule out that such a failure may get reported
-as corrupt input instead of the dedicated error message,
+files. (See the Bugs section of the
+.B /usr/share/doc/lbzip2/README
+file for more on this.) Since the author cannot rule out that such a failure
+may get reported as corrupt input instead of the dedicated error message,
 .B lbzip2
 doesn't reserve a separate exit status for corrupt input -- it could be
 misleading.
